pages/community/ambassador-program/index.js

```diff
@@ -0,0 +1,298 @@
+                    href={`https://www.twitter.com/${ambassador.twitter}`}
+                    target="_blank"
+                    rel="noreferrer"
+                    className="underline"
+                  >
+                    Twitter ↗
+                  </a>
+                  <a
+                    href={`https://www.github.com/${ambassador.github}`}
+                    target="_blank"
+                    rel="noreferrer"
+                    className="underline ml-3"
+                  >
+                    Github ↗
+                  </a>
+                  <a
+                    href={`https://www.linkedin.com/in/${ambassador.linkedin}`}
+                    target="_blank"
+                    rel="noreferrer"
+                    className="underline ml-3"
+                  >
+                    Linkedin ↗
+                  </a>
+                </div>
+              </div>
+            </div>
+          ))}
+        </div>
+      </div>
+      <div className="mt-20">
+        <div className="flex justify-center">
+          <div className="text-center mt-10 lg:w-[55%]">
+            <Heading typeStyle="heading-lg">Tokens of our appreciation</Heading>
+            <Heading
+              typeStyle="body-sm"
+              textColor="text-gray-700"
+              className="text-slate-500 mt-5"
+            >
+              We appreciate and value your commitment and passion to share your
+              knowledge with your communities and we would like to support you
+              in doing so.
+            </Heading>
+          </div>
+        </div>
+        <div className="mt-10">
+          <ul className="grid grid-cols-1 gap-5 sm:grid-cols-3 lg:grid-cols-4">
+            {tokens.map((token) => (
+              <li
+                key={token}
+                className="mt-4 bg-white p-3 shadow-lg rounded-lg flex"
+              >
+                <div>
+                  <div className="bg-pink-200 rounded-full w-[30px] h-[30px] flex flex-col items-center justify-center">
+                    <span className="text-sm">{token.emoji}</span>
+                  </div>
+                </div>
+                <div className="ml-[10px]">
+                  <Heading typeStyle="heading-xs">{token.title}</Heading>
+                  <p className="text-xs mt-[5px] text-slate-600">
+                    {token.details}
+                  </p>
+                </div>
+              </li>
+            ))}
+          </ul>
+        </div>
+        <div className="bg-mapCover bg-dark bg-blend-soft-light bg-center bg-cover w-full h-auto p-4 mt-20 rounded-lg">
+          <div className="flex justify-center">
+            <div className="text-center p-4 text-white lg:w-[65%]">
+              <Heading typeStyle="heading-lg">
+                Become an AsycnAPI Ambassador
+              </Heading>
+              <Heading typeStyle="body-sm">
+                The AsyncAPI Ambassador is now open for nominations! If you’re
```

Comment:
what nominations?

Reply:
Meaning folks can nominate themselves or others to become an ambassador now :)

Reply:
ok, then please make it sound clearer as I was first thinking about some community nominations automatically